src/darwin/mod.rs

```diff
@@ -287,7 +289,20 @@ fn cascade_list_for_languages(ct_font: &CTFont, languages: &[String]) -> Vec<Des
     let list = ct_cascade_list_for_languages(ct_font, &langarr);
 
     // Convert CFArray to Vec<Descriptor>.
-    list.into_iter().map(|fontdesc| Descriptor::new(fontdesc.clone())).collect()
+    list.into_iter()
+        .filter(|fontdesc| unsafe {
+            fontdesc
+                .attributes()
+                .find(kCTFontEnabledAttribute)
+                .and_then(|attr_val| attr_val.downcast::<CFNumber>())
+                .map(|num| match num.to_i32() {
+                    Some(num) => num != 0,
+                    _ => false,
+                })
+                .unwrap_or(true)
```

Comment:
```suggestion
                .and_then(|attr_val| attr_val.downcast::<CFNumber>())
                .and_then(CFNumber::to_i32)
                .map_or(false, |num| num != 0)
```

Could also use `!= Some(0)` in the last line, but I think `map_or` is superior since we're already in a pretty deep iterator chain here.

Also the entire filter could probably be easily moved to a method, seems like something that would be pretty simple to name and document so it likely makes sense.